code/API_definitions/sim-swap-subscriptions.yaml

```
@@ -61,14 +61,14 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: wip
+  version: 0.1.0
   x-camara-commonalities: 0.4.0
 
 externalDocs:
   description: Product documentation at CAMARA
   url: https://github.com/camaraproject/
 servers:
-  - url: '{apiRoot}/sim-swap-subscriptions/v0.1alpha1'
+  - url: '{apiRoot}/sim-swap-subscriptions/v0'

```

API versioning guideline:
```suggestion
  - url: '{apiRoot}/sim-swap-subscriptions/v0.1'

```

---

This one is mandatory to be corrected